core/src/main/scala/scodec/bits/BitVector.scala

```diff
@@ -1013,6 +1016,10 @@ sealed trait BitVector extends BitwiseOperations[BitVector, Long] with Serializa
    */
   final def digest(digest: MessageDigest): BitVector = BitVector(bytes.digest(digest))
 
+  final def cipher(ci: Cipher, key: SecretKeySpec, opmode: Int): BitVector = BitVector(bytes.cipher(ci, key, opmode))
```

Comment:
Can we make this method private? Perhaps this is a silly concern, but I don't like the way JCA uses an Int to represent the opmode.